packages/react-devtools-shared/src/__tests__/ownersListContext-test.js

```diff
@@ -206,4 +206,42 @@ describe('OwnersListContext', () => {
 
     done();
   });
+
+  it('should include all owners for a component wrapped in react memo ', async done => {
+    const InnerComponent = React.forwardRef((props, ref) => <div ref={ref} />);
+    const Memo = React.memo(InnerComponent);
+    const Grandparent = () => {
+      const ref = React.createRef();
+      return <Memo ref={ref} />;
+    };
```

Comment:
nit Let's use a named function so we don't see "Anonymous" in the tests?
```suggestion
    const InnerComponent = (props, ref) => <div ref={ref} />;
    const ForwardRef = React.forwardRef(InnerComponent);
    const Memo = React.memo(ForwardRef);
    const Grandparent = () => {
      const ref = React.createRef();
      return <Memo ref={ref} />;
    };
```